File: leoz-common/src/main/java/org/deku/leoz/rest/entity/zalando/v1/Courier.java
package org.deku.leoz.rest.entity.zalando.v1;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import org.jetbrains.annotations.NotNull;

import java.util.Objects;

@ApiModel(description="Defines the driver who will pick up a tour and deliver all orders related to the tour.")
@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaResteasyServerCodegen", date = "2017-03-10T11:34:55.297Z")
public class Courier   {
  
  private String id = null;
  private String name = null;

  /**
   * Unique identifier of the courier.
   **/
  
  @ApiModelProperty(example = "null", required = true, value = "Unique identifier of the courier.")
  @JsonProperty("id")
  @NotNull
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  /**
   * The name of the courier.
   **/
  
  @ApiModelProperty(example = "null", value = "The name of the courier.")
  @JsonProperty("name")
  public String getName() {
    return name;
  }
  public void setName(String name) {
    this.name = name;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    Courier courier = (Courier) o;
    return Objects.equals(id, courier.id) &&
        Objects.equals(name, courier.name);
  }

  @Override
  public int hashCode() {
    return Objects.hash(id, name);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class Courier {\n");
    
    sb.append("    id: ").append(toIndentedString(id)).append("\n");
    sb.append("    name: ").append(toIndentedString(name)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: leoz-common/src/main/java/org/deku/leoz/rest/entity/zalando/v1/Courier.kt
package org.deku.leoz.rest.entity.zalando.v1

import com.fasterxml.jackson.annotation.JsonProperty
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

import java.util.Objects

@ApiModel(description = "Defines the driver who will pick up a tour and deliver all orders related to the tour.")
@javax.annotation.Generated(value = "io.swagger.codegen.languages.JavaResteasyServerCodegen", date = "2017-03-10T11:34:55.297Z")
class Courier {

    /**
     * Unique identifier of the courier.
     */

    @get:ApiModelProperty(example = "null", required = true, value = "Unique identifier of the courier.")
    @get:JsonProperty("id")
    lateinit var id: String
    /**
     * The name of the courier.
     */

    @get:ApiModelProperty(example = "null", value = "The name of the courier.")
    @get:JsonProperty("name")
    lateinit var name: String


    override fun equals(o: Any?): Boolean {
        if (this === o) {
            return true
        }
        if (o == null || javaClass != o.javaClass) {
            return false
        }
        val courier = o as Courier?
        return id == courier!!.id && name == courier.name
    }

    override fun hashCode(): Int {
        return Objects.hash(id, name)
    }

    override fun toString(): String {
        val sb = StringBuilder()
        sb.append("class Courier {\n")

        sb.append("    id: ").append(toIndentedString(id)).append("\n")
        sb.append("    name: ").append(toIndentedString(name)).append("\n")
        sb.append("}")
        return sb.toString()
    }

    /**
     * Convert the given object to string with each line indented by 4 spaces
     * (except the first line).
     */
    private fun toIndentedString(o: Any?): String {
        if (o == null) {
            return "null"
        }
        return o.toString().replace("\n", "\n    ")
    }
}

